src/SprykerEco/Zed/Computop/Business/Payment/Reader/ComputopPaymentReader.php

```diff
@@ -35,7 +35,7 @@ public function isComputopPaymentExist(QuoteTransfer $quoteTransfer): QuoteTrans
     {
         /** @var \Generated\Shared\Transfer\ComputopCreditCardPaymentTransfer $paymentTransfer */
         $paymentTransfer = $this->getPaymentTransfer($quoteTransfer);
-        $paymentEntity = $this->queryContainer->queryPaymentByTransactionId($paymentTransfer->getTransIdOrFail())->findOne();
+        $paymentEntity = $this->queryContainer->queryPaymentByTransactionId($paymentTransfer->getTransId())->findOne();
```

Comment:
Why was this reverted?

Reply:
earlier this field was not required
so we should not introduce new required properties for existing business logic
I would vote for leaving it as it is

Reply:
I dont see how this query makes sense with null. Wouldnt that Completely make it invalid?

Reply:
my assumption is that we get 1st row from table since typehints are missing
but if null passing leads to exception then we definitely need to require transfer property

@michbeck could you please check behavior for queries when null is passed to `filterBy` propel method?

Reply:
@dereuromark Please see https://github.com/spryker-eco/computop/pull/44#discussion_r781272800

Reply:
Cannot see reference/link, please copy paste relevant info

Reply:
Was reverted because of this:
> looks like breaking change was not required before

You will get first row and no exception. But methods expected string or string[] as parameter and that why this was added at all. Not passing a string is weird behavior imho and only no error is thrown because type hints are missing.

Reply:
So that clarified it, that this is not a bc break but making the error more visible only

Reply:
Fixed